Teleport fails to build on both master branch and release 6.0.2

### Description

**What happened**:
The make full command fails to work

**What you expected to happen**:
Compile the full code

### Reproduction Steps
As minimally and precisely as possible, describe step-by-step how to reproduce the problem.
Reproduction: always
1. Clone the teleport git
2. Run make full inside teleport directory
3. Teleport fails to build

Build log: actually "e" and "webassets" are completely empty

mkdir -p lib/web/build
---> Building OSS web assets.
cd webassets/teleport/ ; zip -qr ../../lib/web/build/webassets.zip .
/bin/sh: riga 1: cd: webassets/teleport/: No such file or directory
zip I/O error: No such file or directory
zip error: Could not create output file (../../lib/web/build/webassets.zip)
make: *** [Makefile:381: lib/web/build/webassets.zip] Errore 15

Latest known working version: teleport 4.1.4

--- a/Makefile
+++ b/Makefile
@@ -376,6 +376,7 @@ tag:
 # build/webassets.zip archive contains the web assets (UI) which gets
 # appended to teleport binary
 $(ASSETS_BUILDDIR)/webassets.zip: | $(ASSETS_BUILDDIR)
+$(ASSETS_BUILDDIR)/webassets.zip: ensure-webassets
 ifneq ("$(OS)", "windows")
 	@echo "---> Building OSS web assets."
 	cd webassets/teleport/ ; zip -qr ../../$@ .

